src/main/java/org/broadinstitute/consent/http/util/HttpClientUtil.java

```diff
@@ -51,6 +51,13 @@ public HttpRequest buildPostRequest(GenericUrl genericUrl, HttpContent content,
     return request;
   }
 
+  public HttpRequest buildUnAuthedPostRequest(GenericUrl genericUrl, HttpContent content) throws Exception {
+    HttpTransport transport = new NetHttpTransport();
+    HttpRequest request = transport.createRequestFactory().buildPostRequest(genericUrl, content);
+    request.setHeaders(new HttpHeaders().set("X-App-ID", "DUOS"));
```

Comment:
No need to change this line - just an explanation. All Terra services communicate with each other and send this header. It's intended to identify the calling app. In this usage, we're talking to Zendesk, so it will likely be ignored.